.github/workflows/build-bootc.yaml

```diff
@@ -32,121 +35,47 @@ jobs:
           popd
 
       - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
+        uses: docker/setup-qemu-action@v3
 
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
+        uses: docker/setup-buildx-action@v3
 
-      - name: Login to Quay.io
+      - name: Login to registry.redhat.io (for RHEL image)
         uses: docker/login-action@v3
```

Comment:
This should have an `if {{ $matrix.flavor == rhel }}` conditional So that you only do it on that variant.